core/src/main/java/io/github/belgif/rest/guide/validator/core/parser/Parser.java

```diff
@@ -250,6 +252,7 @@ private void parsePaths(SourceDefinition sourceDefinition, ParserResult result)
         if (paths == null) {
             return;
         }
+        result.pathsDefinitions.add(new PathsDefinition(paths, sourceDefinition.getFileName()+"/paths", openApiFile, result));
```

Comment:
the identifier can just be fixed to "paths" in the PathsDefinition constructor.  I don't think it'll be really used, and doesn't contain the file name normally.

Reply:
In case of multiple PathsDefinitions (usefull when paths in external files are references) there will be multiple PathsDefinitions with the same identifier.

Reply:
The identifier doesn't need to be unique across files; like already the case for other definition types.